tests/utils.py

```
@@ -12,6 +13,21 @@
 
 
+def has_ipv6(host: str) -> bool:
+    sock = None
+    ipv6_enabled = False
+    if socket_module.has_ipv6:
+        try:
+            sock = socket_module.socket(socket_module.AF_INET6)
+            sock.bind((host, 0))
+            ipv6_enabled = True
+        except Exception:  # pragma: no cover
+            pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)